src/createAction.js

```diff
@@ -6,17 +6,20 @@ export default function createAction(type, actionCreator, metaCreator) {
   const finalActionCreator = typeof actionCreator === 'function'
     ? actionCreator
     : identity;
+
   const actionHandler = (...args) => {
+    const hasError = args[0] instanceof Error;
```

Comment:
Can you use `if (action.payload instanceof Error)` [here](https://github.com/acdlite/redux-actions/blob/master/src/createAction.js#L19-L22) instead of `args[0]` please?